atc/db/check_factory.go

```diff
@@ -139,8 +139,20 @@ func (c *checkFactory) Resources() ([]Resource, error) {
 	var resources []Resource
 
 	rows, err := resourcesQuery.
-		Join("(select DISTINCT(resource_id) FROM job_inputs) ji ON ji.resource_id = r.id").
-		Where(sq.Eq{"p.paused": false}).
+		LeftJoin("(select DISTINCT(resource_id) FROM job_inputs) ji ON ji.resource_id = r.id").
+		LeftJoin("(select DISTINCT(resource_id) FROM job_outputs) jo ON jo.resource_id = r.id").
+		Where(sq.Or{
+			sq.And{
+				sq.Eq{"p.paused": false},
+				sq.Eq{"r.active": true},
```

Comment:
Pretty sure we want `r.active` to be `true` all the time - I believe 'active' means 'still configured', so if it's `false` that means someone removed it from their pipeline config. It seems safer to put that back on `resourcesQuery` since it seems like a safe assumption that we only ever want to deal with still-configured resources. (Unless I'm missing some edge case?)